backend/app/router/endpoints/notes.py

```
@@ -0,0 +1,252 @@
+        2. All other components receive the output of the previous step
+    3.  The list of vectors gets passed into the self attention layer
+    4. Then the output gets passed into the feed forward neural network."""
+
+    summary = """
+A Transformer is a type of Machine Learning Model Architecture.
+
+It can be trained to predict the most probable output given an input.
+
+Outputs and Inputs are determined by the dataset used during training of the model.
+
+Currently (2024), the most used applications of the transformer model are Large Language Models. (GPT, BART, ect.)"""
```

Pylint (3.3.7)

[convention] Line too long (117/100)

(C0301)